source/apiVolontaria/apiVolontaria/models.py

```diff
@@ -86,3 +86,23 @@ def expire(self):
         """Expires a token by setting its expiration date to now."""
         self.expires = timezone.now()
         self.save()
+
+
+class Profile(models.Model):
+    user = models.OneToOneField(
+        User,
+        on_delete=models.CASCADE,
+        related_name='profile',
+    )
+    phone = models.CharField(
+        verbose_name="Phone number",
+        max_length=15,
+        null=True,
+        blank=True,
+    )
```

Comment:
Aren't there any predefined phone number format? We should probably use a regular expression here to be sure a correct number is sent to the DB. Have a look at this [Stack Overflow answer](https://stackoverflow.com/a/19131360) giving some guidelines for a fail-safe phone number field.